tests/test_save_load.py

```diff
@@ -5,7 +5,7 @@
 from pandas.testing import assert_frame_equal, assert_series_equal
 
 import cleanvision
-from cleanvision.imagelab import Imagelab
+from cleanvision import Imagelab
```

Comment:
```suggestion
from cleanvision.imagelab import Imagelab
```

Reply:
how about we keep the old way of importing for here, to ensure backwards compatibility in the future?

Reply:
We can do that